### ISSUE TITLE
`--config` flag is broken

### ISSUE THREAD

**Issue by irmiller22** [igorshubovych/markdownlint-cli#173]
Hi! It seems like the latest release broke the way the `--config` flag is handled?

I use the `--config` flag in my CI/CD workflow for markdown linting, and as of a few hours ago it's no longer respecting the configuration that I've set up.

UPDATE:

Adding additional context with an example:

```
# .github/markdownlint.yaml

MD007:
  indent: 4
MD013: false
MD040: false
MD046: false
```

```bash
markdownlint -c .github/markdownlint.yaml .
```

The settings specified in `.github/markdownlint.yaml` were being respected until a few hours ago, which I imagine coincides with the 0.27.0 release.

**Comment by DavidAnson** [comment]
See also https://github.com/DavidAnson/markdownlint/issues/373

I'll have a look tonight.

**Comment by DavidAnson** [comment]
Can you please give an example that does not work? All the tests are passing, so it should not be completely broken?

**Comment by irmiller22** [comment]
Sure, I'll also add it to the initial comment:

```
# .github/markdownlint.yaml

MD007:
  indent: 4
MD013: false
MD040: false
MD046: false
```

```bash
markdownlint -c .github/markdownlint.yaml .
```

The settings specified in `.github/markdownlint.yaml` were being respected until a few hours ago, which I imagine coincides with the 0.27.0 release.

**Comment by DavidAnson** [comment]
Thanks, I can reproduce this. Looking now...

**Comment by DavidAnson** [comment]
I should have a fix shortly.

**Comment by DavidAnson** [comment]
Please try `v0.27.1', thanks, and sorry for the trouble!

**Comment by irmiller22** [comment]
@DavidAnson no worries, thank you for the fix!
